### DISCUSSION TITLE
如何关掉候选词选单中，词库带出的拼音

### DISCUSSION THREAD

**Discussion by marcushsu** [Mintimate/oh-my-rime#106] [Q&A] [answered]
我不太懂输入法的专业术语，请看图，就是关掉红框的部分：
![image](https://github.com/Mintimate/oh-my-rime/assets/11764515/33a1ff33-f58f-4442-b385-918a261d89a1)
这个最夸张，超出我屏幕之外了：
![image](https://github.com/Mintimate/oh-my-rime/assets/11764515/00e3a2ed-10b4-4391-86bd-fc55c13d6ffe)

**Comment by YummyCocoa** [comment]
你这个看着估计是Linux，而且 librime 的版本不够高的原因。 导致自动纠错的 Lua 没有生效的样子。

librime 如果版本足够：

<img width="217" alt="image" src="https://github.com/Mintimate/oh-my-rime/assets/132128788/923f7f8a-099c-4bcd-8b9c-dc9b4ef1cc7c">


参考这个部分： https://www.mintimate.cc/zh/guide/faQ.html#linux%E8%96%84%E8%8D%B7%E9%85%8D%E7%BD%AE%E6%97%A0%E6%B3%95%E4%BD%BF%E7%94%A8

不过最好的方法，估计还是自己手动编译安装 ibus-rime(带上librime-lua插件)

**Reply by marcushsu** [reply]
我这个是Windows，小狼毫0.16.1。

**Comment by marcushsu** [comment, marked as answer]
终于在[这里](https://github.com/rime/home/discussions/1366)找到了答案。把`rime_mint.schema.yaml`里的`spelling_hints`和`always_show_comments`两个关掉。

**Reply by YummyCocoa** [reply]
奇怪，我在Windows上的小狼毫是不会这样的 ：
![测试结果](https://github.com/Mintimate/oh-my-rime/assets/132128788/d73df4bf-7a41-4cfe-9fe9-2535105f386c)

rime_mint.schema.yaml 里设置spelling_hints和always_show_comments，是因为配合自动纠错Lua。理论上实际使用，在输入字的时候，会被自动纠错把正确读音的拼音过滤掉，你是不会看到这些的。也就是，使用效果：
 
![自动纠错使用效果](https://github.com/Mintimate/oh-my-rime/assets/132128788/7dce6ccd-965e-43d9-8e7b-184da7f58223)

你这种情况，你可以这样：

1. 更新小狼毫到最新每夜版本；
2. 同步用户词典，之后把小狼毫服务关闭，删除小狼毫的build文件夹；
3. 重启小狼毫服务，重新部署并同步用户词典。

之后看看有没有这种问题把。

**Reply by marcushsu** [reply]
我测试的结果，问题又复现。
我的测试步骤：
1. 停掉小狼毫；
2. 编辑`rime_mint.schema.yaml`，将关掉的`spelling_hints`和`always_show_comments`再打开；
3. 将`Build`文件夹更名为`Build.bak`；
4. 下载Nightly Build并安装，之后重新开机。
